frontend/packages/yki/src/pages/AccessibilityStatementPage.tsx

```diff
@@ -44,16 +44,18 @@ const ItemBulletList = ({
   item: string;
   bulletPoints: Array<string>;
 }) => (
-  <Typography variant="body1" component="div">
-    <ul className="accessibility-statement-page__item-bullet-list">
-      <li>{item}</li>
-      <ul>
-        {bulletPoints.map((bulletPoint, i) => (
-          <li key={`${bulletPoint}-${i}`}>{bulletPoint}</li>
-        ))}
-      </ul>
+  <>
+    <Typography variant="body1" component="li">
+      {item}
+    </Typography>
+    <ul>
+      {bulletPoints.map((bulletPoint, i) => (
+        <Typography key={`${bulletPoint}-${i}`} variant="body1" component="li">
+          {bulletPoint}
+        </Typography>
+      ))}
     </ul>
```

Comment:
```suggestion
    <Typography variant="body1" component="li">
      {item}
      <ul>
        {bulletPoints.map((bulletPoint, i) => (
          <Typography key={`${bulletPoint}-${i}`} variant="body1" component="li">
            {bulletPoint}
          </Typography>
        ))}
      </ul>
     </Typography>
```

Tässä kai pitäis tuo sisempi `<ul>` tulla tuon `<li>` sisälle. Nyt käy niin, että on sisäkkäin `<ul><ul>...`

Reply:
Joo, totta. Fiksaan.